taxonomies/additives.txt

```diff
@@ -23245,12 +23744,46 @@ wikipedia:en: https://en.wikipedia.org/wiki/Steviol_glycoside
 
 en: E960a, Steviol glycosides from Stevia
 xx: E960a
+ar: E960a, جليكوسيدات ستيفيول من ستيفيا
+az: E960a, Steviyadan Steviol qlikozidləri
+bg: E960a, Стевиол гликозиди от Стевия
+ca: E960a, Glicòsids d'esteviol procedents d'estèvia
+cs: E960a, Steviol glykosidy ze Stévie
+da: E960a, Steviolglycosider fra Stevia
 de: E960a, Steviolglycoside aus Stevia
+el: E960a, Γλυκοσίδες στεβιόλης από Στέβια
 es: E960a, Glucósidos de esteviol procedentes de estevia
+et: E960a, Steviolglükosiidid Steviast
+fi: E960a, Stevioliglykosidit Steviasta
 fr: E960a, Glycosides de stéviol issus de Stevia
+he: E960a, גליקוזידים סטיביול מסטיביה
+hi: E960a, स्टीविया से स्टीविओल ग्लाइकोसाइड
+hr: E960a, Steviol glikozidi iz Stevije
+hu: E960a, Steviol-glikozidok a Steviából
+id: E960a, Steviol glikosida dari Stevia
 it: E960a, Glicosidi steviolici da Stevia
+ja: E960a, ステビアからのステビオール配糖体
+ko: E960a, 스테비아에서 추출한 스테비올 배당체
+lt: E960a, Steviolio glikozidai iš Stevia
+lv: E960a, Steviola glikozīdi no Stevia
+mk: E960a, Стевиол гликозиди од Стевиа
+ms: E960a, Steviol glikosida daripada Stevia
+mt: E960a, Glikosidi ta' stevjol minn Stevia
 nl: E960a, Steviolglycosiden uit Stevia
+pl: E960a, Glikozydy stewiolowe ze stewii
```

Comment:
2025-03-07T21:50:07.0406507Z ERROR - duplicate_synonym - 25523 - pl:glikozydy-stewiolowe-ze-stewii already is a synonym of pl:e960 for entry en:e960 (ingredients) - pl:glikozydy-stewiolowe-ze-stewii cannot be mapped to entry en:e960a / pl:e960a

Reply:
fixed, I don't quite understand the diff between E960 and E960a by the way 🙃